test/evergreen.yml

```diff
@@ -1246,12 +1246,8 @@ tasks:
           dependent_task: compile-linux-no-ftruncate
       - func: "unit test"
 
-  # Break out Python unit tests into multiple buckets/tasks based on test name and runtime
-  # The test/suite/run.py script can work out test names by casting each command argument
-  # with "test_" prefix and "*.py" postfix.
-  #
-  # One example:
-  # "test/suite/run.py [ab]" will be translated to testing "test_a*.py" and "test_b*.py"
+  # Break out Python unit tests into multiple buckets/tasks.  We have 11 buckets,
```

Comment:
"11 buckets" is just what it was extended to (from the initial "8 buckets") last time. May worth leaving the specific number out to make future bucket extension/shrinking work easier?

Reply:
I've updated the comment.